op-node/node/client.go

```diff
@@ -189,6 +221,11 @@ func (p *PreparedL1Endpoint) Setup(ctx context.Context, log log.Logger, rollupCf
 	return p.Client, sources.L1ClientDefaultConfig(rollupCfg, p.TrustRPC, p.RPCProviderKind), nil
 }
 
+func (p *PreparedL1Endpoint) SetupBlobClient(ctx context.Context, log log.Logger) ([]client.RPC, error) {
+	// TODO add test
+	return nil, nil
```

Comment:
Please check if this code will cause UT or E2E to fail. It is best to include test code as part of the formal code to avoid having to fix it later.